node.js

```diff
@@ -416,14 +416,21 @@ module.exports = {
     if (process.env.BROWSERSLIST_IGNORE_OLD_DATA) return
 
     var latest = latestReleaseTime(agentsObj)
-    var halfYearAgo = Date.now() - TIME_TO_UPDATE_CANIUSE
+    var currentTime = Date.now()
+
+    if (latest !== 0 && latest < currentTime - TIME_TO_UPDATE_CANIUSE) {
```

Comment:
`1000 * 60 * 60 * 24 * 30` math is dublicated

Reply:
You can get month from last update and then compare it with `> 6` and not use `TIME_TO_UPDATE_CANIUSE`